### DISCUSSION TITLE
[To :] in Backend grids

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1742] [Questions & Answers] [answered]
Does anyone have an idea why in all the grids in Backend it was chosen to place the punctuation mark in the case of the **To** field by putting a space after the word? It does not look good at all and stands out immediately. If it were up to me I would give up the punctuation mark because it is a form of visual abuse that is inherited from the ancient times.

See bellow one example:

![screenshot](https://user-images.githubusercontent.com/8360474/126807333-439f8235-4095-4739-9a83-cb4538c45869.png)

**Comment by luigifab** [comment]
For me it's an error. In english there are no spaces for "To:", but in french there is a space for "From :" and "To :" translations.

**Reply by fballiano** [reply]
I noticed it now after more than 10 years 😅 
For sure it was a mistake

**Reply by fballiano** [reply, marked as answer]
it was too easy, I've created a quick PR 😅😅😅

https://github.com/OpenMage/magento-lts/pull/1755

**Reply by addison74** [reply]
@fballiano - I really didn't realize it for 12 years either. But once you realize this issue it scratches your retina 😅

**Reply by fballiano** [reply]
@ADDISON74 exactly! :-D

**Reply by addison74** [reply]
I think the Magento team left us an Easter Egg. If I bow my head to the right and look better as if I see Santa with a pipe in his mouth 😅

**Comment by addison74** [comment]
The colon appears on the date columns. No correction is needed because I will come up with a PR to remove these colons because they look  ugly (From, To, In). 

![to_image](https://user-images.githubusercontent.com/8360474/173693533-06bb1683-7eac-4e3e-9c63-cd146a91f5c2.jpg)

Files responsible for this issue (unmodified yet): 

/app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Filter/Date.php
/app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Filter/Datetime.php

In addition, the calendar button will be after the input box not at the beginning (another visual issue).
